rate_limiter/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "rate_limiter"
+version = "0.1.0"
+authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
```

Comment:
I think we should change it simply because I don't think the FC team want emails about some crate that we've created. Is `authors` even a mandatory field? i.e. could we simply remove it.

Reply:
Cool. The `authors` filed is not needed unless we are publishing it to `crate.io`. Let's simply remove it now.